go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/mdehoog/op-rosetta
 go 1.18
 
 require (
-	github.com/coinbase/rosetta-geth-sdk v0.0.4
-	github.com/coinbase/rosetta-sdk-go v0.7.11
+	github.com/coinbase/rosetta-geth-sdk v0.0.5-0.20230124171712-bed30e6be8c7
```

Comment:
should we do a release rather than relying on the sha?

Reply:
last time, we modified our sdk according to an asset very frequently, if we keep releasing new versions, perhaps there will be a lot of future releases, but I am ok to both choice.

Reply:
@mdehoog not this sdk, we modified an internal sdk according to an blockchain very frequently